spec/features/idv/steps/gpo_otp_verification_step_spec.rb

```diff
@@ -19,59 +23,42 @@
     )
   end
   let(:user) { profile.user }
-
-  it 'prompts for one-time code at sign in' do
-    sign_in_live_with_2fa(user)
-
-    expect(current_path).to eq idv_gpo_verify_path
-    expect(page).to have_content t('idv.messages.gpo.resend')
-
-    gpo_confirmation_code
-    fill_in t('forms.verify_profile.name'), with: otp
-    click_button t('forms.verify_profile.submit')
-
-    expect(user.events.account_verified.size).to eq 1
-    expect(page).to_not have_content(t('account.index.verification.reactivate_button'))
-  end
-
-  it 'renders an error for an expired GPO OTP' do
-    sign_in_live_with_2fa(user)
-
-    gpo_confirmation_code.update(code_sent_at: 11.days.ago)
-    fill_in t('forms.verify_profile.name'), with: otp
-    click_button t('forms.verify_profile.submit')
-
-    expect(current_path).to eq idv_gpo_verify_path
-    expect(page).to have_content t('errors.messages.gpo_otp_expired')
-
-    user.reload
-
-    expect(user.events.account_verified.size).to eq 0
-    expect(user.active_profile).to be_nil
+  let(:threatmetrix_enabled) { false }
+  let(:threatmetrix_review_status) { nil }
+
+  before do
+    allow(IdentityConfig.store).to receive(:lexisnexis_threatmetrix_enabled).
+      and_return(threatmetrix_enabled)
+    allow(IdentityConfig.store).to receive(:lexisnexis_threatmetrix_required_to_verify).
+      and_return(threatmetrix_enabled)
+    allow(IdentityConfig.store).to receive(:proofing_device_profiling_decisioning_enabled).
+      and_return(threatmetrix_enabled)
   end
 
-  it 'allows a user to resend a letter' do
-    allow(Base32::Crockford).to receive(:encode).and_return(otp)
-
-    sign_in_live_with_2fa(user)
-
-    expect(GpoConfirmation.count).to eq(0)
-    expect(GpoConfirmationCode.count).to eq(0)
-    click_on t('idv.messages.gpo.resend')
+  it_behaves_like 'gpo otp verification'
```

Comment:
I pulled these out into a shared example:

- `prompts for one-time code at sign in`
- `renders an error for an expired GPO OTP`
- `allows a user to resend a letter`

The idea is that all the standard GPO OTP verification stuff should be unaffected by ThreatMetrix status: if you fail ThreatMetrix, you can still verify your account, but afterwards you'll have to take steps to fully activate.